src/SFA.DAS.EmployerAccounts.Web/Extensions/AgreementTemplateExtensions.cs

```diff
@@ -19,19 +20,22 @@ public static string InsetText(this AgreementTemplateViewModel agreementTemplate
                 case 3:
                     return "This is a new agreement.";
                 case 4:
-                    bool v3Accepted = GetV3AgreementStatus(organisationAgreementViewModel);
-                    return v3Accepted ? "This is a variation to the agreement we published 9 January 2020. You only need to accept it if you want to access incentive payments for hiring a new apprentice." : "This is a new agreement.";
+                case 5:
+                    bool v3orv4Accepted = GetV3orV4AgreementStatus(organisationAgreementViewModel);
+                    return v3orv4Accepted ? "This is a variation to the agreement we published 9 January 2020. You only need to accept it if you want to access incentive payments for hiring a new apprentice." : "This is a new agreement.";
                 default:
                     return string.Empty;
             }
         }
 
-        private static bool GetV3AgreementStatus(ICollection<OrganisationAgreementViewModel> organisationAgreementViewModel)
+        private static bool GetV3orV4AgreementStatus(ICollection<OrganisationAgreementViewModel> organisationAgreementViewModel)
```

Comment:
Next time there is another agreement will this become `GetV3OrV4OrV5AgreementStatus()`

Reply:
Modified code